baselines/bigquery-storage/src/v1beta1/big_query_storage_client.ts.baseline

```diff
@@ -217,6 +218,13 @@ export class BigQueryStorageClient {
       const callPromise = this.bigQueryStorageStub.then(
         stub => (...args: Array<{}>) => {
           if (this._terminated) {
+            if (methodName in this.descriptors.stream) {
+              const stream = new PassThrough();
+              setImmediate(() => {
+                stream.emit('error', new GoogleError('The client has already been closed.'));
+              });
+              return stream;
+            }
```

Comment:
doesn't this need to also happen in the rejection arm? ie if the `bigQueryStorageStub` is rejected (ie. no creds), the the promise will still get bubbled up to streaming.js

Reply:
If the `bigQueryStorageStub` is rejected, it throw an error(see line 225 below). This wouldn't get bubble up to streaming.js.

In other words, only when`bigQueryStorageStub` resolved, and client was terminated, the streaming call would receive `Passthrough` stream with error event.

Reply:
ok